.circleci/config.yml

```diff
@@ -170,11 +170,19 @@ jobs:
             if ! ( [ -d ~/.ideprobe-uitests/artifacts ] && [ -n "$(ls -A ~/.ideprobe-uitests/artifacts)" ] ); then exit 0; fi
             for f in $(ls -d ~/.ideprobe-uitests/artifacts/*/) ; do cd  $f ; zip -r "../$(basename ${f}).zip" . ; done
             mkdir -p ~/artifacts/zip
+            mkdir -p ~/artifacts/logs
             find ~/.ideprobe-uitests/artifacts -name '*.zip' | xargs cp --target-directory ~/artifacts/zip
+            for f in $(find ~/.ideprobe-uitests/idea-logs/intellij-instance-*/logs/idea.log) ; do
+              target="~/artifacts/logs/$(echo $f | sed 's/.*\/\(intellij-instance-.*\)\/logs\/idea.log/\1.log/')"
+              eval "cp $f $target"
+            done
           when: on_fail
       - store_artifacts:
           path: ~/artifacts/zip
           destination: .
+      - store_artifacts:
```

Comment:
@PawelLipski does it make sense to include whole `~/artifacts` in the artifacts?

Reply:
Hmmm what else can be there?

Reply:
Other than logs? Also screenshots right?

Reply:
yup, here is what we have if ci failes:
https://app.circleci.com/pipelines/github/VirtusLab/git-machete-intellij-plugin/6488/workflows/76db5f9d-0931-497e-b91d-47f8ab8b1199/jobs/6866/artifacts
and here, when it passes:
https://app.circleci.com/pipelines/github/VirtusLab/git-machete-intellij-plugin/6489/workflows/dbbeb06d-248a-48c7-b133-225c0e02f86b/jobs/6867/artifacts